contents/founders/cracked-manifesto.md

```diff
@@ -0,0 +1,88 @@
+---
+date: 2024-03-16
+title: A cracked manifesto
+author:
+  - charles-cook
+featuredImage: ../images/blog/transparency.jpeg
+featuredImageType: full
+tags:
+  - Culture
+  - People
+crosspost:
+  - Blog      
+---
+
+We’ve become a bit obsessed with the notion of hiring cracked people at PostHog. When we say cracked, we mean things like:
+
+- Taking extreme ownership of ideas and driving them to completion, even if others disagree with you. Some people call this founder mentality. Better still, you bring others on the journey with you.
+ 
+- Unwavering optimism in the face of change or new ideas. This doesn’t mean you have to agree with the change, but adopting a ‘yes and’ mentality helps new ideas get off the ground that might otherwise die in a committee somewhere. This is especially true for ideas that come from non-senior/new people.
```

Comment:
```suggestion
- Unwavering optimism in the face of change or new ideas. This doesn’t mean you have to agree with the change, but adopting a ‘yes and’ mentality helps new ideas get off the ground that might otherwise die in a committee somewhere. This is especially true for ideas that come from non-senior or new people.
```